### PR TITLE
openshot: update to 2.4.4

### PR DESCRIPTION
**Pull request by luzpaz**
Release notes: https://github.com/OpenShot/openshot-qt/releases/tag/v2.4.4

### CODE REVIEW

File: srcpkgs/openshot/template [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # Template file for 'openshot'
 pkgname=openshot
-version=2.4.3
-revision=2
```
Comment (maxice8) [inline srcpkgs/openshot/template]: reset revision to `1`, don't delete it

**Comment by maxice8** [comment]
Tou also need to update libopenshot and libopenshot-audio

**Comment by maxice8** [comment]
Sorry but this is not in acceptable condition.

- One commit per package with a proper commit message
- Look at travis and fix the build errors

**Comment by luzpaz** [comment]
No idea what this means: https://travis-ci.org/void-linux/void-packages/jobs/514775879#L2676-L2685

```
=> ERROR: libopenshot-0.2.3_1: SONAME bump detected: libopenshot.so.16 -> libopenshot.so.17
=> ERROR: libopenshot-0.2.3_1: please update common/shlibs with this line: "libopenshot.so.17 libopenshot-0.2.3_1"
=> ERROR: libopenshot-0.2.3_1: all reverse dependencies should also be revbumped to be rebuilt against libopenshot.so.17:
=> ERROR: libopenshot-0.2.3_1: cannot continue with installation!
=> ERROR: libopenshot-0.2.3_1: pre-pkg_99-pkglint: 'grep -E "${_pattern}" $mapshlibs' exited with 1
=> ERROR:   in hook() at common/hooks/pre-pkg/99-pkglint.sh:108
=> ERROR:   in run_func() at common/xbps-src/shutils/common.sh:21
=> ERROR:   in run_pkg_hooks() at common/xbps-src/shutils/common.sh:254
=> ERROR:   in main() at common/xbps-src/libexec/xbps-src-prepkg.sh:47
The command "common/travis/build.sh $BOOTSTRAP $ARCH" exited with 1.
```
What is `common/shlibs` ?

**Comment by maxice8** [comment]
On Sun, Apr 07, 2019 at 05:44:52AM -0700, luzpaz wrote:
> No idea what this means: https://travis-ci.org/void-linux/void-packages/jobs/514775879#L2676-L2685
> 
> ```
> => ERROR: libopenshot-0.2.3_1: SONAME bump detected: libopenshot.so.16 -> libopenshot.so.17
> => ERROR: libopenshot-0.2.3_1: please update common/shlibs with this line: "libopenshot.so.17 libopenshot-0.2.3_1"
> => ERROR: libopenshot-0.2.3_1: all reverse dependencies should also be revbumped to be rebuilt against libopenshot.so.17:
> => ERROR: libopenshot-0.2.3_1: cannot continue with installation!
> => ERROR: libopenshot-0.2.3_1: pre-pkg_99-pkglint: 'grep -E "${_pattern}" $mapshlibs' exited with 1
> => ERROR:   in hook() at common/hooks/pre-pkg/99-pkglint.sh:108
> => ERROR:   in run_func() at common/xbps-src/shutils/common.sh:21
> => ERROR:   in run_pkg_hooks() at common/xbps-src/shutils/common.sh:254
> => ERROR:   in main() at common/xbps-src/libexec/xbps-src-prepkg.sh:47
> The command "common/travis/build.sh $BOOTSTRAP $ARCH" exited with 1.
> ```
> What is `common/shlibs` ?
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/10487#issuecomment-480587091

See common/shlibs file

**Comment by maxice8** [comment]
> No idea what this means: https://travis-ci.org/void-linux/void-packages/jobs/514775879#L2676-L2685
> 
> ```
> => ERROR: libopenshot-0.2.3_1: SONAME bump detected: libopenshot.so.16 -> libopenshot.so.17
> => ERROR: libopenshot-0.2.3_1: please update common/shlibs with this line: "libopenshot.so.17 libopenshot-0.2.3_1"
> => ERROR: libopenshot-0.2.3_1: all reverse dependencies should also be revbumped to be rebuilt against libopenshot.so.17:
> => ERROR: libopenshot-0.2.3_1: cannot continue with installation!
> => ERROR: libopenshot-0.2.3_1: pre-pkg_99-pkglint: 'grep -E "${_pattern}" $mapshlibs' exited with 1
> => ERROR:   in hook() at common/hooks/pre-pkg/99-pkglint.sh:108
> => ERROR:   in run_func() at common/xbps-src/shutils/common.sh:21
> => ERROR:   in run_pkg_hooks() at common/xbps-src/shutils/common.sh:254
> => ERROR:   in main() at common/xbps-src/libexec/xbps-src-prepkg.sh:47
> The command "common/travis/build.sh $BOOTSTRAP $ARCH" exited with 1.
> ```
> 
> What is `common/shlibs` ?

It tells you to update the entry in common/shlibs, replacing libopenshot.so.16 with libopenshot.so.17, also putting the proper version.

**Comment by maxice8** [comment]
There should be 3 separate commits

- `libopenshot-audio: update to 0.1.8` -> This is where you update `libopenshot-audio` and change its common/shlibs entry
- `libopenshot: update to 0.2.3` -> This is where you update `libopenshot` and change its common/shlibs entry
- `openshot: update to 2.4.4` -> This is where you update `openshot`

**Comment by luzpaz** [comment]
sorry, I was planning to attend to this. Thank you for redoing the commits.

**Comment by maxice8** [comment]
No worries, please check the commits to see if they have the proper authorship

**Pull request closed without merging**